netflix-no-ipv6-dns-proxy.init

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh -e
+
+### BEGIN INIT INFO
+# Provides:          netflix-no-ipv6-dns-proxy
+# Required-Start:    $remote_fs
+# Required-Stop:     $remote_fs
+# Should-Start:      $network $syslog
+# Should-Stop:       $network $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: Start and stop netflix-no-ipv6-dns-proxy
+# Description:       Fix the Netflix anti-v6 madness with v6 tunnel providers
+### END INIT INFO
+
+PATH=/sbin:/bin:/usr/sbin:/usr/bin
+
+# for a chrooted server: "-u bind -t /var/lib/named"
+# Don't modify this line, change or create /etc/default/bind9.
+OPTIONS=""
+
+. /lib/lsb/init-functions
+
+case "$1" in
+    start)
+	log_daemon_msg "Starting Netflix domain name service fix..." "netflix-no-ipv6-dns-proxy"
+
+	modprobe capability >/dev/null 2>&1 || true
+
+	# dirs under /var/run can go away on reboots.
+	if start-stop-daemon --start -b --oknodo --quiet --exec /opt/netflix-no-ipv6-dns-proxy/server.py \
+		-- $OPTIONS; then
+	    log_end_msg 0
+	else
+	    log_end_msg 1
+	fi
+    ;;
+
+    stop)
+	log_daemon_msg "Stopping Netflix domain name service fix..." "netflix-no-ipv6-dns-prox"
+
+	pid=$(ps ax|grep server.py |grep python | awk '{print $1}')
```

Comment:
I'm a bit worried that this has the potential to kill other processes.  Perhaps you could use the `--make-pidfile /var/run/netflix-no-ipv6-dns-proxy.pid` option on line 30 to create a pid file, then instead of using ps/kill directly here, use `start-stop-daemon --stop --pidfile /var/run/netflix-no-ipv6-dns-proxy.pid --remove-pidfile`?